package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mini-dashboard",
-  "version": "0.2.17",
+  "version": "0.2.18",
```

Comment:
Can we create a different PR for the version upgrade? 🙏

Reply:
We want to be fast on this one since the engine team is waiting + timezone issue between Laurent and us, so I would go with this even if it's not really clean, I agree 😊

Reply:
Thanks for the feedback, I will keep that in mind for the future.